addon/components/scatter-chart.js

```diff
@@ -145,7 +161,10 @@ const ScatterChartComponent = ChartComponent.extend(LegendMixin, FloatingTooltip
     var xDomain = this.get('xDomain');
     var graphicLeft = this.get('graphicLeft');
     var graphicWidth = this.get('graphicWidth');
-    var padding = (xDomain[1] - xDomain[0]) * this.get('graphPadding');
+    var padding = 0;
+    if (this.get('hasXDomainPadding')) {
```

Comment:
I agree with billy, it makes sense to have a default value and the secondary value only occurs under certain conditions, and we'd still have to define it at the top #hoisting #notcoffeescript